src/scenic/syntax/compiler.py

```diff
@@ -76,6 +82,21 @@ def generic_visit(self, node):
             )
         return super().generic_visit(node)
 
+    def visit(self, node: ast.AST | list[ast.AST]):
```

Comment:
Because `generateInvocation` returns a list of AST nodes, we need to change the `visit` to support list